planner_cspace/src/planner_3d.cpp

```diff
@@ -1243,6 +1246,9 @@ class Planner3dNode
     escaping_ = false;
     cnt_stuck_ = 0;
 
+    diag_updater_.setHardwareID("none");
+    diag_updater_.add("Status", this, &Planner3dNode::diagnoseStatus);
```

Comment:
It looks better to be bit more detailed like "Path Planner Status". (I'm not very familiar to ROS diagnostics manner.)

Reply:
Yes, more detailed info is better. We can easily remove the prefix "Path Planner" in parameter file when we run diagnostic_aggregator node to publish /diagnostics_agg , if it does not necessary.

e.g.
```
planner:
  type: GenericAnalyzer
  path: Path Planner
  find_and_remove_prefix: "planner_3d: Path Planner"
```
![selection_215](https://user-images.githubusercontent.com/6742467/46387405-87ada900-c701-11e8-8c77-eea5db1d2c07.png)